psa-crypto/src/types/key.rs

```diff
@@ -770,25 +747,110 @@ pub struct Policy {
 #[derive(Copy, Clone, Debug, Default, PartialEq, Serialize, Deserialize, Zeroize)]
 pub struct UsageFlags {
     /// Permission to export the key.
-    pub export: bool,
+    export: bool,
     /// Permission to copy the key.
-    pub copy: bool,
+    copy: bool,
     /// Permission for the implementation to cache the key.
-    pub cache: bool,
+    cache: bool,
     /// Permission to encrypt a message with the key.
-    pub encrypt: bool,
+    encrypt: bool,
     /// Permission to decrypt a message with the key.
-    pub decrypt: bool,
+    decrypt: bool,
     /// Permission to sign a message with the key.
-    pub sign_message: bool,
+    sign_message: bool,
     /// Permission to verify a message signature with the key.
-    pub verify_message: bool,
+    verify_message: bool,
     /// Permission to sign a message hash with the key.
-    pub sign_hash: bool,
+    sign_hash: bool,
     /// Permission to verify a message hash with the key.
-    pub verify_hash: bool,
+    verify_hash: bool,
     /// Permission to derive other keys from this key.
-    pub derive: bool,
+    derive: bool,
+}
+
+impl UsageFlags {
```

Comment:
I have two suggestions/simplifications for the setters/getters which are up for discussion:
- since everything is set to a secure default (`false`), I think one would only use `set_...(true)`, right? Because of that, maybe we could have the `set_...` methods not taking a `bool` parameter but setting their flag to `true`?
- I have seen this before in other code, and maybe we are doing it somewhere, where the getters do not have the `get_` prefix. For example `export` instead of `get_export` would return the `export` flag. Would that be ok?